libs/pilight/core/common.c

```diff
@@ -1019,3 +1024,161 @@ int file_get_contents(char *file, char **content) {
 	fclose(fp);
 	return 0;
 }
+
+int checkdnsrr(const char *domain, const char *type)
+{
+	logprintf(LOG_STACK, "%s(%s,%s)", __FUNCTION__, domain ? domain : "(NULL)", type ? type : "(NULL)");
+	return 0;	// NYI.
+}
+
+#if defined(__FreeBSD__) || defined(_WIN32)
+#define use_regexp 0
+#else
+#define use_regexp 1
+#endif
+
+// Do NOT write the next as range A-Za-z0-9 because the string may be used without RE.
+#define AtoZ_atoz_0to9 "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
```

Comment:
>  Finally I wanted it to look more similar to the RE range-expression that would be written inside [ ]

That's what we need to avoid. Having defines look similar as the rest.